crates/wash-cli/src/call.rs

```diff
@@ -182,6 +180,10 @@ pub struct CallCommand {
     /// Fully qualified WIT export to invoke on the component, e.g. `wasi:cli/run.run`
     #[clap(name = "function")]
     pub function: String,
+
+    /// Input to the function, in a format that is hopefully readable by the function
+    #[clap(long = "input")]
+    pub input: Option<String>,
```

Comment:
I don't see this used anywhere, can we remove it?

Reply:
Ah sorry forgot to update, but this is fixed -- I expanded this to be a bit more inline with what I wanted, more customization for people using `wash call`